utils/test-utils/src/main/java/com/microsoft/device/dualscreen/utils/test/TestSyncUtils.kt

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+package com.microsoft.device.dualscreen.utils.test
+
+import java.util.concurrent.CountDownLatch
+import java.util.concurrent.TimeUnit
+
+/**
+ * Simple implementation for [ScreenInfoListener] that saves internally the last screen info data.
+ */
+class TestSyncUtils {
```

Comment:
`TestSync` utils makes me think about a `sync` mechanism like calling to a network API and so on. Maybe we can change the name? 
Also its header comment is outdated referring to `ScreenInfoListener`